package.json

```diff
@@ -14,7 +14,7 @@
     "redis": "redis-server /usr/local/etc/redis.conf",
     "prod-server": "bundle exec foreman start -f Procfile",
     "ci": "start-server-and-test prod-server http://localhost:5000 test-ci",
-    "lint": "eslint --ext js --ext jsx .",
+    "lint": "eslint --ext js --ext jsx",
```

Comment:
VIM trying to lint the whole project instead of the files that were opened 🙃 vscode works fine with this config too, but if there's any problem we can go back to the prev version.